packages/gatsby-cli/src/reporter/__tests__/index.ts

```diff
@@ -83,15 +83,19 @@ describe(`report.error`, () => {
     const generatedError = getErrorMessages(
       reporterActions.createLog as jest.Mock
     )[0]
-    expect(generatedError).toMatchSnapshot()
```

Comment:
these snapshots contained full stack traces down to the node.js level which weren't stable across node.js versions. it didn't seem like that was material to the tests, so we just simplified this.